src/main/java/muramasa/gti/common/Data.java

```diff
@@ -58,51 +60,123 @@ public static void init() {
     public static ItemBasic MotorMV = new ItemBasic(Ref.ID, "motor_mv");
     public static ItemBasic MotorHV = new ItemBasic(Ref.ID, "motor_hv");
     public static ItemBasic MotorEV = new ItemBasic(Ref.ID, "motor_ev");
     public static ItemBasic MotorIV = new ItemBasic(Ref.ID, "motor_iv");
+    public static ItemBasic MotorLUV = new ItemBasic(Ref.ID, "motor_luv");
+    public static ItemBasic MotorZPM = new ItemBasic(Ref.ID, "motor_zpm");
+    public static ItemBasic MotorUV = new ItemBasic(Ref.ID, "motor_uv");
+    public static ItemBasic MotorUHV = new ItemBasic(Ref.ID, "motor_uhv");
+    public static ItemBasic MotorUEU = new ItemBasic(Ref.ID, "motor_ueu");
+
+    public static ItemBasic PumpULV = new ItemBasic(Ref.ID, "pump_ulv", "160 L/s (as Cover)");
     public static ItemBasic PumpLV = new ItemBasic(Ref.ID, "pump_lv", "640 L/s (as Cover)");
     public static ItemBasic PumpMV = new ItemBasic(Ref.ID, "pump_mv", "2,560 L/s (as Cover)");
     public static ItemBasic PumpHV = new ItemBasic(Ref.ID, "pump_hv", "10,240 L/s (as Cover)");
     public static ItemBasic PumpEV = new ItemBasic(Ref.ID, "pump_ev", "40,960 L/s (as Cover)");
     public static ItemBasic PumpIV = new ItemBasic(Ref.ID, "pump_iv", "163,840 L/s (as Cover)");
+    public static ItemBasic PumpLUV = new ItemBasic(Ref.ID, "pump_luv", "655,360 L/s (as Cover)");
+    public static ItemBasic PumpZPM = new ItemBasic(Ref.ID, "pump_zpm", "2,621,440 L/s (as Cover)");
+    public static ItemBasic PumpUV = new ItemBasic(Ref.ID, "pump_uv", "10,485,760 L/s (as Cover)");
+    public static ItemBasic PumpUHV = new ItemBasic(Ref.ID, "pump_uhv", "41,943,040 L/s (as Cover)");
+    public static ItemBasic PumpUEU = new ItemBasic(Ref.ID, "pump_ueu", "167,772,160 L/s (as Cover)");
+
+    public static ItemBasic Solar = new ItemBasic(Ref.ID, "solar", "1 EU/t (as Cover)");
+    public static ItemBasic SolarULV = new ItemBasic(Ref.ID, "solar_ulv", "8 EU/t (as Cover)");
+    public static ItemBasic SolarLV = new ItemBasic(Ref.ID, "solar_lv", "32 EU/t (as Cover)");
+    public static ItemBasic SolarMV = new ItemBasic(Ref.ID, "solar_mv", "128 EU/t (as Cover)");
+    public static ItemBasic SolarHV = new ItemBasic(Ref.ID, "solar_hv", "512 EU/t (as Cover)");
+    public static ItemBasic SolarEV = new ItemBasic(Ref.ID, "solar_ev", "2048 EU/t (as Cover)");
+    public static ItemBasic SolarIV = new ItemBasic(Ref.ID, "solar_iv", "8192 EU/t (as Cover)");
+    public static ItemBasic SolarLUV = new ItemBasic(Ref.ID, "solar_luv", "32,768 EU/t (as Cover)");
+    public static ItemBasic SolarZPM = new ItemBasic(Ref.ID, "solar_zpm", "131,072 EU/t (as Cover)");
+    public static ItemBasic SolarUV = new ItemBasic(Ref.ID, "solar_uv", "524,288 EU/t (as Cover)");
+    public static ItemBasic SolarUHV = new ItemBasic(Ref.ID, "solar_uhv", "2,097,152 EU/t (as Cover)");
+    public static ItemBasic SolarUEU = new ItemBasic(Ref.ID, "solar_ueu", "8,388,608 EU/t (as Cover)");
```

Comment:
Solars as covers should not have anything higher than MV imo, it will be too OP.

The tooltips will need to change too since they won't operate at a constant EU/t but will change based on weather, exposure and/or time.